include/LLVMSPIRVOpts.h

```diff
@@ -65,6 +65,13 @@ enum class ExtensionID : uint32_t {
   Last,
 };
 
+enum class BIsRepresentation : uint32_t {
+  OpenCL12,
+  OpenCL20
+  // TODO: consider targeting SPIR-V friendly IR for some non-OpenCL backends,
```

Comment:
Wouldn't this also help testing / debug ?

Reply:
Added one more option for that